### DISCUSSION TITLE
error when installing Moose

### DISCUSSION THREAD

**Discussion by tqcthai** [idaholab/moose#26220] [Q&A General]
Dear all,

I've installed Moose and got this error message:

MOOSE_projects/moose/framework/src/userobjects/RadialAverage.C:254:14: error: 'SearchParams' is not a member of 'nanoflann'; did you mean 'SearchParameters'?
  254 |   nanoflann::SearchParams search_params;
      |              ^~~~~~~~~~~~
      |              SearchParameters

/MOOSE_projects/moose/framework/src/userobjects/RadialAverage.C:262:69: error: 'search_params' was not declared in this scope; did you mean '_ri_params'?
  262 |         &(bn(0)), Utility::pow<2>(_radius + _padding), ret_matches, search_params);
      |                                                                     ^~~~~~~~~~~~~
      |                                                                     _ri_params

I wonder how to fix it?

Thanks.
T

**Comment by GiudGiud** [comment]
Hello

Is this a new install from scratch? Or you updated moose (and forgot to update libmesh conda packages?) ?

Guillaume

**Reply by tqcthai** [reply]
Hi Giudgiud,

It is a new install.

Thai

**Reply by tqcthai** [reply]
I checked conda list | grep libmesh, and the moose-libmesh is 2023.11.29

**Comment by loganharbour** [comment]
Your libmesh is up to date but your MOOSE is out of date. The line that you're erroring on was changed in https://github.com/idaholab/moose/commit/445e4bf158d987ab662a73482c3afd4d1c663b77, more than a month ago.
